diff --git a/Makefile.am b/Makefile.am
index d9cfbf6e..acf7f8fe 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -211,8 +211,6 @@ shairport_sync_mpris_test_client_SOURCES = shairport-sync-mpris-test-client.c
 shairport_sync_mpris_test_client_LDADD = lib_mpris_interface.a
 endif
 
-if INSTALL_CONFIG_FILES
-
 CONFIG_FILE_INSTALL_TARGET = config-file-install-local
 $(CONFIG_FILE_INSTALL_TARGET): scripts/shairport-sync.conf
 	install -d $(DESTDIR)$(sysconfdir)
@@ -257,8 +255,6 @@ $(MPRIS_POLICY_INSTALL_TARGET): $(MPRIS_POLICY_FILE)
 
 endif # USE_MPRIS
 
-endif # INSTALL_CONFIG_FILES
-
 INSTALL_GROUP_TARGET = install-group-local
 
 $(INSTALL_GROUP_TARGET):
@@ -329,10 +325,16 @@ $(INSTALL_CYGWIN_TARGET): scripts/shairport-sync-config
 
 endif # INSTALL_CYGWIN_SERVICE
 
-install-exec-hook: $(CONFIG_FILE_INSTALL_TARGET) \
+install-config-files: $(CONFIG_FILE_INSTALL_TARGET) \
                    $(DBUS_POLICY_INSTALL_TARGET) \
                    $(MPRIS_POLICY_INSTALL_TARGET) \
                    $(INSTALL_SYSTEMV_TARGET) \
                    $(INSTALL_SYSTEMD_TARGET) \
                    $(INSTALL_FREEBSD_TARGET) \
                    $(INSTALL_CYGWIN_TARGET)
+
+if INSTALL_CONFIG_FILES
+
+install-exec-hook: install-config-files
+
+endif
